crossplane-contrib/provider-styra
# [WIP] Upd examples

### Description of your changes
- Some examples do not seem to reflect the current API, this PR should correct that.


### Fixes
- Divergence of examples with code
- Correct examples
- Some typos

I have:


### How has this code been tested
Actually no code change happened. But the examples files were tested like so:

- Change `example-kubernetes-v2-system.yaml` according to the local/cluster setup.
- ```bash
  $ kubectl apply -f examples/system/example-kubernetes-v2-system.yaml
  $ kubectl apply -f examples/config/example-provider-config.yaml
  $ make run
  ```
- Check for errors
- Exam Styra instance



[contribution process]: https://git.io/fj2m9
